examples/requirements.txt

```
@@ -5,5 +5,6 @@ git+https://github.com/ax3l/PyNAFF.git@fix-complex-warning
 matplotlib
 numpy
 openpmd-api
+pals-schema
```

I wonder if I should write
```suggestion
pals-schema~=0.1
```
because we will release and break often :D